File: app/src/main/java/com/example/learnenglishwordssecondtry/ChooseSetToViewActivity.java
package com.example.learnenglishwordssecondtry;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class ChooseSetToViewActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_choose_set_to_view);

        Button learnedWordsButton;
        Button wordsInProcessButton;

        learnedWordsButton = findViewById(R.id.showLearnedWordsButton);
        wordsInProcessButton = findViewById(R.id.showWordsInProcessButton);

        learnedWordsButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                /*Intent intent = ViewWordsActivity.newIntent(
                        ChooseSetToViewActivity.this, true);
                startActivity(intent);*/
                Toast.makeText(ChooseSetToViewActivity.this, "This option isn't available", Toast.LENGTH_SHORT).show();

            }
        });

        wordsInProcessButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = ViewWordsActivity.newIntent(
                        ChooseSetToViewActivity.this, false);
                startActivity(intent);
            }
        });


    }
}

File: app/src/main/java/com/example/learnenglishwordssecondtry/viewChangeWords/ChooseSetToViewActivity.kt
package com.example.learnenglishwordssecondtry.viewChangeWords

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.example.learnenglishwordssecondtry.R
import com.example.learnenglishwordssecondtry.viewChangeWords.wordsListActivity.WordsListActivity

class ChooseSetToViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_choose_set_to_view)

        val btnViewWordsLearned: Button = findViewById(R.id.showLearnedWordsButton)
        val btnViewWordsInProcess: Button = findViewById(R.id.showWordsInProcessButton)

        btnViewWordsLearned.setOnClickListener {
            Toast.makeText(this@ChooseSetToViewActivity, "This option isn't available", Toast.LENGTH_SHORT).show()
        }

        btnViewWordsInProcess.setOnClickListener {
            val intent: Intent = WordsListActivity().newIntent(this@ChooseSetToViewActivity, false)
            startActivity(intent)
        }

    }

}